examples/heartbeat_demo/doctor.py

```diff
@@ -25,15 +26,19 @@
 # Boring setup stuff #
 ######################
 
-SEEDNODE_URL = 'localhost:11501'
+try:
```

Comment:
Perhaps a comment here to remind the demo operator whats up with teachers :+1:

Reply:
Sorry, what _is_ up with teachers?
You mean something like "if you are running this demo with a local fleet, the default will work fine"?

Reply:
Yeah - Since this script can be used with an optional CLI argument, it will be helpful to describe it's usage here.  Perhaps will help calm a future on-stage presenter :-)